src/jni_core/src/jvm_runtime.rs

```diff
@@ -123,6 +123,7 @@ impl JavaVmWrapper {
             .version(JNIVersion::V8)
             .option("-Dis_embedded_connector=true")
             .option(format!("-Djava.class.path={}", class_vec.join(":")))
+            .option("--add-opens=java.base/java.nio=org.apache.arrow.memory.core,ALL-UNNAMED")
```

Comment:
What's this use for?

Reply:
https://arrow.apache.org/docs/java/install.html

snowflake iceberg sdk